Framework/PythonInterface/plugins/algorithms/LoadElementalAnalysisData.py

```diff
@@ -0,0 +1,184 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+
+import glob
+import os
+import numpy as np
+from mantid import config
+from mantid.api import AlgorithmFactory, AnalysisDataService, PythonAlgorithm, TextAxis, WorkspaceGroup, WorkspaceGroupProperty
+from mantid.kernel import Direction, IntBoundedValidator
+from mantid.simpleapi import ConvertToHistogram, ConvertToPointData, DeleteWorkspace, LoadAscii, WorkspaceFactory
+
+
+TYPE_KEYS = {"10": "Prompt", "20": "Delayed", "99": "Total"}
+SPECTRUM_INDEX = {"Delayed": 1, "Prompt": 2, "Total": 3}
+NUM_FILES_PER_DETECTOR = 3
+RUN_NUMBER_LENGTH = 5
+
+
+class LoadElementalAnalysisData(PythonAlgorithm):
+    def category(self):
+        """
+        Returns category
+        """
+        return "DataHandling"
+
+    def PyInit(self):
+        self.declareProperty(name='Run', defaultValue=0, validator=IntBoundedValidator(lower=0),
+                             doc='Run number to load')
+
+        self.declareProperty(WorkspaceGroupProperty(name='GroupWorkspace', defaultValue='',
+                                                    direction=Direction.Output),
+                             doc='Output group workspace for run')
+
+        self.declareProperty(name="directory", defaultValue="", direction=Direction.Output,
```

Comment:
Can we call this property `Directory`? We try to be consistent and have all property names use Pascal Case. It looks like the RHEL system test failure has picked this one up.